secure/key-manager/src/logging.rs

```diff
@@ -14,6 +15,8 @@ pub struct LogSchema<'a> {
     consensus_key: Option<&'a Ed25519PublicKey>,
     json_rpc_endpoint: Option<&'a str>,
     #[schema(display)]
+    key_manager_config: Option<&'a KeyManagerConfig>,
+    #[schema(display)]
```

Comment:
You can save yourself the Display above, by doing `#[schema(debug)]`

Reply:
Aah, nice! Didn't know we supported using the debug format. Let's go with that 😄